src/main/java/com/fcgl/madrid/points/service/UserTrophyService.java

```diff
@@ -2,21 +2,41 @@
 
 import com.fcgl.madrid.points.model.UserTrophy;
 import com.fcgl.madrid.points.repository.UserTrophyRepository;
+import com.fcgl.madrid.points.model.InternalStatus;
+import com.fcgl.madrid.points.model.StatusCode;
+import com.fcgl.madrid.points.model.ExceptionUtil;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.TransactionSystemException;
 
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+
+import java.lang.Exception;
+import java.lang.Throwable;
+import java.lang.StringBuilder;
+
+import java.util.Set;
 import java.util.List;
+import java.util.ArrayList;
 
 @Service
 public class UserTrophyService {
-    private UserTrophyRepository userTrophyRepository;
+	private UserTrophyRepository userTrophyRepository;
+
+	@Autowired
+	public UserTrophyService(UserTrophyRepository userTrophyRepository) {
+		this.userTrophyRepository = userTrophyRepository;
+	}
+
+	public List<UserTrophy> findAll() {
+		return userTrophyRepository.findAll();
+	}
 
-    @Autowired
-    public UserTrophyService(UserTrophyRepository userTrophyRepository) {
-        this.userTrophyRepository = userTrophyRepository;
-    }
+	public List<UserTrophy> getUserTrophiesById(Long userId) {
```

Comment:
Missing `@CircuitBreaker(name = "backendA", fallbackMethod = "fallback")` above this function and the fallback method is needed in this class as well

Reply:
You Should be returning something similar to `GetUserPointsResponse` here.  Maybe `GetUserTrophiesResponse` with `InternalStatus` and `List<UserTrophy>`